src/install.ml

```diff
@@ -213,21 +223,18 @@ module Entry = struct
   let set_src t src = { t with src }
 
   let relative_installed_path t ~paths =
-    let main_dir = Section.Paths.get paths t.section in
-    let dst =
-      match t.dst with
-      | Some x -> x
-      | None ->
-        let dst = Path.basename t.src in
-        match t.section with
-        | Man -> begin
-            match String.rsplit2 dst ~on:'.' with
-            | None -> dst
-            | Some (_, sec) -> sprintf "man%s/%s" sec dst
```

Comment:
As far as I remember, @emillon added this code to fix the installation of man pages. Let's see what he has to say.

Reply:
If I'm reading the code correctly, exactly the same logic is duplicated in `Section.Paths.install_path` which calls `man_subdir` which does something similar.

So if this is a behavior change, it's not intended to be so.

Reply:
These changes were moved out to #2105